django/db/models/functions/datetime.py

```diff
@@ -142,6 +151,12 @@ class ExtractSecond(Extract):
 ExtractYear.register_lookup(YearLt)
 ExtractYear.register_lookup(YearLte)
 
+ExtractIsoYear.register_lookup(YearExact)
```

Comment:
I don't see any test failures if these are commented out. Further, the `iso_year` lookup should be documented in `docs/ref/models/querysets.txt` and in the release notes.